src/components/VoiceRecorder/VoiceRecorder.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Mic from '@material-ui/icons/Mic';
+import './VoiceRecorder.css';
+
+export default class VoiceRecorder extends React.Component {
+  state = {
+    record: false,
+    mediaRecorder: '',
+    chunks: '',
+    audioURL: '',
+    iconsColor: 'black'
+  };
+
+  render() {
```

Comment:
By convention we put `render` methods last in class, all of our components follow this, so it'll be odd for developers to see it up above.

You should place it as the last method in the class